themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -65,73 +74,119 @@ region=Estado
 postal_code=CEP
 country=Pa\u00EDs
 emailVerified=E-mail verificado
-gssDelegationCredential=gss delega\u00E7\u00E3o credencial
+gssDelegationCredential=Delega\u00E7\u00E3o de Credenciais GSS
+
+profileScopeConsentText=Perfil de usu\u00E1rio
+emailScopeConsentText=Endere\u00E7o de e-mail
+addressScopeConsentText=Endere\u00E7o
+phoneScopeConsentText=N\u00FAmero de telefone
+offlineAccessScopeConsentText=Acesso Offline
+samlRoleListScopeConsentText=Minhas Fun\u00E7\u00F5es
```

Comment:
I don't have a better translation for roles :) But `funções` sounds odd. Maybe `Meus Perfis de Acesso` ?

Reply:
Yes, much better